app/policies/application_policy.rb

```diff
@@ -0,0 +1,49 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    true
+  end
+
+  def show?
+    true
+  end
+
+  def create?
+    user.present?
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    return true if user.present? && user.admin?
```

Comment:
It is application policy, as usual this leaves true of false by default and rewrite it in needed places